data/json/snippets/e_logs.json

```diff
@@ -203,6 +203,46 @@
+      },
+      {
+        "id": "local_files_simple_52",
+        "text": "Scrolling through the device’s video storage, you come across an archived video message received, apparently, through a chat app a few days after the first mass-spread portal storms.  Evidently taped through the low-grade camera of a smartphone, the footage shows the interior view of a small room bathed in darkness, a door visible in the background with several heavy-duty dead bolts drawn, and the grime-caked face of a balding man hovering in the foreground. Quivering like a particularly well-bearded leaf, the man's gaze continually darts around as he mutters into the microphone in a horse cadence, barely above a whisper.\n\n\"Denis, I don’t have a clue if you and Lorence are still alive; fuck knows, after the shit that’s been going on, I don’t know how anyone can still be alive,  but, in the name of hell and Jesus Christ, I hope you are, because you need to come get me, man; you really, really need to come get me out of here.  Something's gotten into my house.  I don’t know what it wants; it hasn’t hurt me yet, but it’s not leaving, it’s not letting me leave, it just watches me all day and all night, watching and following.  I managed to lock myself in the broom closet.  I don’t think it can get in, but you need to come get me.  Please.  Please, for the love of God.  Send somebody.\"\n\nAs the man’s babbling trails onwards, your eyes are increasingly drawn to the space over the door, where a slight quivering in the wall is steadily growing more pronounced.  Suddenly, yellowed, rat-like incisors burst through the space, furiously and swiftly gnawing a softball-sized hole through the wood.  Noiselessly withdrawing, the fangs’ place is taken by a disturbingly slender arm that limply drapes itself through the aperture, the unnaturally long hand at its end feeling along the wall with fingers that tangle like the leaves of a spider plant.  The cameraman only grows wise to the spaghetti-thin limb’s intrusion as the fingers lightly cascade over his shoulders, moments before the arm entwines about him with tentacle-like deftness.\n\nThe view’s sent spiraling to the ground as the limb constricts and whips him rearwards, a sickening series of crunches and pained wails audible as the man’s slammed against the heavy door with machine gun rapidity.  The last few shots consist of the man being yanked through a freshly splintered hole in the door as, upside down, a horrifically bloated head slithers down the wall.  Domed like the cranium of an infant, sparsely furred, inset with anglerfish-like eyes, and far too large for the ragged straw of flesh that composes its neck, the being’s skull passively observes the blooded, screaming mess of a body that it’s trying to feed through the watermelon-sized gap.  It succeeds… eventually."
```

Comment:
```suggestion
        "text": "Scrolling through the device’s video storage, you come across an archived video message received, apparently, through a chat app a few days after the first mass-spread portal storms.  Evidently taped through the low-grade camera of a smartphone, the footage shows the interior view of a small room bathed in darkness, a door visible in the background with several heavy-duty dead bolts drawn, and the grime-caked face of a balding man hovering in the foreground.  Quivering like a particularly well-bearded leaf, the man's gaze continually darts around as he mutters into the microphone in a horse cadence, barely above a whisper.\n\n\"Denis, I don’t have a clue if you and Lorence are still alive; fuck knows, after the shit that’s been going on, I don’t know how anyone can still be alive,  but, in the name of hell and Jesus Christ, I hope you are, because you need to come get me, man; you really, really need to come get me out of here.  Something's gotten into my house.  I don’t know what it wants; it hasn’t hurt me yet, but it’s not leaving, it’s not letting me leave, it just watches me all day and all night, watching and following.  I managed to lock myself in the broom closet.  I don’t think it can get in, but you need to come get me.  Please.  Please, for the love of God.  Send somebody.\"\n\nAs the man’s babbling trails onwards, your eyes are increasingly drawn to the space over the door, where a slight quivering in the wall is steadily growing more pronounced.  Suddenly, yellowed, rat-like incisors burst through the space, furiously and swiftly gnawing a softball-sized hole through the wood.  Noiselessly withdrawing, the fangs’ place is taken by a disturbingly slender arm that limply drapes itself through the aperture, the unnaturally long hand at its end feeling along the wall with fingers that tangle like the leaves of a spider plant.  The cameraman only grows wise to the spaghetti-thin limb’s intrusion as the fingers lightly cascade over his shoulders, moments before the arm entwines about him with tentacle-like deftness.\n\nThe view’s sent spiraling to the ground as the limb constricts and whips him rearwards, a sickening series of crunches and pained wails audible as the man’s slammed against the heavy door with machine gun rapidity.  The last few shots consist of the man being yanked through a freshly splintered hole in the door as, upside down, a horrifically bloated head slithers down the wall.  Domed like the cranium of an infant, sparsely furred, inset with anglerfish-like eyes, and far too large for the ragged straw of flesh that composes its neck, the being’s skull passively observes the blooded, screaming mess of a body that it’s trying to feed through the watermelon-sized gap.  It succeeds… eventually."
```